src/instructlab/model/evaluate.py

```diff
@@ -301,8 +301,8 @@ def launch_server(
             logger.warning(
                 f"Based on your hardware configuration, when using vLLM, we recommend setting max-workers between {recommended_min_workers} and {recommended_max_workers} for optimal performance"
             )
-        gpus = gpus or eval_serve.vllm.gpus
-        if gpus:
+        is_gpus = gpus or eval_serve.vllm.gpus
```

Comment:
I don't think this is right.  gpus is used below and it needs the updated value.

Reply:
Ack - looking more closely at this code now - what is the behavior if both `gpus` and `eval_serve.vllm.gpus` are both set to positive integers?

Reply:
gpus wins over vllm.gpus